src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    ? people.filter(p => p.sex === 'm'
```

Comment:
Forgot the parentheses for start of the function
```suggestion
    ? people.filter((p) => p.sex === 'm'
```